client/extensions/zoninator/state/locks/reducer.js

```diff
@@ -0,0 +1,67 @@
+/** @format */
+
+/**
+ * Internal dependencies
+ */
+import { combineReducers, createReducer, keyedReducer } from 'state/utils';
+import {
+	ZONINATOR_REQUEST_LOCK,
+	ZONINATOR_REQUEST_LOCK_ERROR,
+	ZONINATOR_UPDATE_LOCK,
+} from '../action-types';
+
+const isRequesting = createReducer(
+	{},
```

Comment:
Since this reducer generally holds a bool, it should be initialized with one. Pretty safe to assume that we aren't requesting upon app startup, so `false` should be fine.